tools/sigmut/sigmut.xml

```diff
@@ -0,0 +1,495 @@
+					<param name="vcfile" value="icgc"/>
+					<param name="icgc_file" ftype="txt" value="test_matrix.txt"/>
+				</conditional>
+				<conditional name="bed_input">
+					<param name="bedfile" value="no"/>	
+				</conditional>
+				<param name="plot" value="True"/>
+			</conditional>
+
+		<output name="SBS6" file="SBS_6.pdf" lines_diff="5"/>
+		<output name="SBS24" file="SBS_24.pdf" lines_diff="5"/>
+		<output name="SBS78" file="SBS_78.pdf" lines_diff="5"/>
+		<output name="SBS96" file="SBS_96.pdf" lines_diff="5"/>
+		<output name="SBS384" file="SBS_384.pdf" lines_diff="5"/>
+		<output name="SBS1536" file="SBS_1536.pdf" lines_diff="5"/>
+		<output name="DBS78" file="DBS_78.pdf" lines_diff="5"/>
+		<output name="DBS186" file="DBS_186.pdf" lines_diff="5"/>
+
+		</test>
+
+
+		<test>
+                        <conditional name="set_analysis">
+                                <param name="choices" value="get_sigmut"/>
+                                <param name="refgendat" ftype="fasta" value="GRCh37"/>
+                                <conditional name="vcfile_input">
+                                        <param name="vcfile" value="icgc"/>
+                                        <param name="icgc_file" ftype="txt" value="test_matrix.txt"/>
+                                </conditional>
+                                <conditional name="bed_input">
+                                        <param name="bedfile" value="no"/>
+                                </conditional>
+                                <param name="tsb_stat" value="True"/>
+                        </conditional>
+
+		<output name="sigRes" file="significantResults_strandBiasTest.txt" />
+		<output name="TSB24" file="strandBiasTest_24.txt"/>
+		<output name="TSB384" file="strandBiasTest_384.txt"/>
+		<output name="TSB6144" file="strandBiasTest_6144.txt"/>
+		</test>
+
+	</tests>
+
+	<help><![CDATA[
+
+		**SigProfiler**
+
+		This script configures the SigProfiler analysis pipeline.
+		You must specify a VCF file for at least one sample.
+
+
+		**Options**
+		--version             	show program's version number and exit
+
+		-h, --help            	show this help message and exit
+
+		--install_genome	Install de novo any of the following reference
+					genomes: 'GRCh37', 'GRCh38', 'mm9' or 'mm10'.
+
+		--name=APPENDIX		Provide a project name
+
+		--genome=NAME		Provide a reference genome (ex: GRCh37, GRCh38,
+					mm9 or mm10).
+
+		--files=Abs_path	Path where the input vcf files are located
+
+		--exome			Use only the exome or not
+
+		--bed=FILE		BED file containing the set of regions to be used
+					in generating the matrices
+
+		--chrom			Create the matrices on a per chromosome basis
+
+		--plot			Generate the plots for each context
+
+		--tsb			Performs a transcriptional strand bias test for the
+					24, 384, and 6144 contexts
+
+		--gs			Performs a gene strand bias test
+
+		For further info see: https://github.com/AlexandrovLab/SigProfilerMatrixGenerator
```

Comment:
This tool is important. It deserves a more detailed and pedagogical help section, since it is targeted to clinician researcher.

In addition, I definitely think that the tool should be associated with the description of a real test case in a compagnon pdf with a url in the help section.

Reply:
Ok, I will modify the help section for a better explanation. Afterwards, I will work in a real example that can be reached by a URL link.

Reply:
The explanation on the file names that you provided is good to integrate in the help section !